Module-6-Assignment/ClassUpdationTriggerTest.cls

```diff
@@ -0,0 +1,23 @@
+/**
+* Test Class for testing ClassUpdationTrigger.
+*/
+@isTest
+public class ClassUpdationTriggerTest {
+    
+    @isTest
+    private static void testStudentDeletion()
```

Comment:
change the method name to properly depict what it is doing

Reply:
Done